### DISCUSSION TITLE
Help with filtering releases

### DISCUSSION THREAD

**Discussion by noobworth** [ImranR98/Obtainium#1788] [Q&A] [answered]
How can I tell Obtainium to ignore "webview" releases from [this](https://github.com/ungoogled-software/ungoogled-chromium-android/releases) repo?
I know about the "Filter Release Titles by Regular Expression" field, but I don't know much about RegEx and the solutions I found online didn't work so I'm asking here.

Edit:
Tried using `-webview$` in both "Filter Release Titles by Regular Expression" and "Trim Version String With RegEx" but the APK selector still shows me the apk's from the latest "webview" release:
![Screenshot_2024-08-14-23-44-47-341_dev.imranr.obtainium.jpg](https://github.com/user-attachments/assets/77d39eae-bf05-448b-a61f-543ee13e0129)

**Comment by DarkCrypt** [comment, marked as answer]
That repo is dead.

This will ignore the webview releases...
Filter Release Titles by Regular Expression: `^((?!.*webview.*).)*$`

You can also filter apks with Filter APKs by Regular Expression so you don't need to use the selector.

**Reply by noobworth** [reply]
> That repo is dead.

Yeah I only noticed this after posting the question. Still the answer will be helpful for filtering other repos. Thanks!

**Comment by Truinto** [comment]
I was going crazy why this would not work for me. I just found that I had to scroll down and put the regex into "filter APK" instead of "filter release title". Looking at the app's release page I can see the difference. Leaving this here for other people.
